python/ray/__init__.py

```diff
@@ -19,6 +19,16 @@
     os.path.abspath(os.path.dirname(__file__)), "pickle5_files")
 sys.path.insert(0, pickle5_path)
 
+# Expose ray related symbols which may be dependent by other shared
+# liraries such as _streaming.so. Only linux needs this.
+from os.path import dirname
+so_path = os.path.join(dirname(__file__), "_raylet.so")
+if os.path.exists(so_path):
```

Comment:
Is this `if` needed?

Reply:
windows use `dll`, and doesn't need to expose symbols, because it always expose symbols